packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -191,10 +193,12 @@ export class RunningSummarizer implements IDisposable {
             : undefined;
 
     public handleSystemOp(op: ISequencedDocumentMessage) {
+        this.numSystemOps++;
```

Comment:
This code (separation of handleSystemOp & handleOp looks wrong). Can you please change it to use just one handler? I'd suggest to listen on IDeltaManager's "op" event, but using IDeltaManager.inbound's "op" event is fine too.